src/main/java/HomeWork11/Main.java

```diff
@@ -0,0 +1,43 @@
+package HomeWork11;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        String[] words = {"привет" , "пока", "кот", "пес", "сергей", "узкий", "пока", "зачем", "кто", "кот"};
+
+        Map<String, Integer> wordsMap = new HashMap<>();
+
+        for (String word : words){
+
+            if (!wordsMap.containsKey(word)){
+                wordsMap.put(word, 1);
+            } else {
+                int curValue = wordsMap.get(word);
+                wordsMap.put(word, ++curValue);
+            }
+
+        }
+
+        for(Map.Entry<String, Integer> item : wordsMap.entrySet()){
+
+            System.out.printf("Слово: %s Повторений: %d \n", item.getKey(), item.getValue());
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.